packages/store-ui/src/atoms/Input/Input.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import type { InputProps } from './Input'
+import Root from './Input'
+
+export default {
+  title: 'Atoms/Input',
+} as Meta
+
+const InputTemplate: Story<InputProps> = (args) => <Root {...args} />
+
+export const Input = InputTemplate.bind({})
+Input.args = {
+  children: 'Input',
+}
```

Comment:
`children` for inputs 🤔? Wouldn't `value` or `placeholder` be better?

Reply:
Oh, my bad. I just copy/past the button story. I'll fix it!